src/main/java/io/confluent/copycat/hdfs/partitioner/DailyPartitioner.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ **/
+
+package io.confluent.copycat.hdfs.partitioner;
+
+import org.joda.time.DateTimeZone;
+
+import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import io.confluent.common.config.ConfigException;
+import io.confluent.copycat.hdfs.HdfsSinkConnectorConfig;
+
+public class DailyPartitioner extends TimeBasedPartitioner {
+
+  private static long partitionDurationMs = TimeUnit.HOURS.toMillis(24);
+  private static String pathFormat = "'daily'/YYYY/MM/dd/";
```

Comment:
I'm confused about what this string is doing. Is the `'daily'` supposed to be in there like that? And have the tick marks around it? Is that just joda time formatting? And if that's prefixing with a `daily` directory, is that what users would actually want or do they only want the `YYYY/MM/dd` part?